soak_bench_scripts/soak.sh

```diff
@@ -1,7 +1,8 @@
-#!/bin/bash
+#!/usr/local/bin/bash
+echo $BASH_VERSION
```

Comment:
Was this maybe just needed for debugging and can be removed?

Reply:
Fair point - relates to the above. I'll take it out.